internal/evaluator/criteria.go

```diff
@@ -48,6 +50,8 @@ func (c *Criteria) addItem(key, value string) {
 		if c.digestItems == nil {
 			c.digestItems = make(map[string][]string)
 		}
+		// the image url can contain https://
+		key = cleanKey(key)
```

Comment:
How?

Reply:
I allowed users to add the component/image url with a `https://` prefix. But now that I think about it, I'm not sure that adds any value. I may remove that option.